app/js/index.controller.js

```diff
@@ -0,0 +1,16 @@
+angular
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["angular" is not defined.](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2321393357/issues/source?bid=3196969&fileBranchId=3199483#l1)